docs/buildrun.md

```diff
@@ -293,6 +293,7 @@ status:
     git:
       commitAuthor: xxx xxxxxx
       commitSha: f25822b85021d02059c9ac8a211ef3804ea8fdde
+      branchName: main
```

Comment:
May you add a quick level-setting sentence above ... instead of

> Example of a `BuildRun` with surfaced results for `git` source:

we take:

> Example of a `BuildRun` with surfaced results for `git` source (note that the branchName is only included if the Build does not specify any revision):